ebos/eclmpiserializer.hh

```diff
@@ -64,7 +64,7 @@ decltype(auto) make_variant(std::size_t index)
 }
 
 template<class T>
-using remove_cvr_t = std::remove_const_t<std::remove_reference_t<T>>;
+using remove_cvr_t = std::remove_volatile_t<std::remove_const_t<std::remove_reference_t<T>>>;
```

Comment:
For what it's worth, `std::remove_cv_t<>` removes both `const` and `volatile`.

Reply:
simplified.